uptane-whitepaper.md

```diff
@@ -1 +1,106 @@
+Though not directed against automobiles, the recent [SolarWinds](https://en.wikipedia.org/wiki/SolarWinds)hack in which companies, government agencies, 
+and academic institutions suffered significant data breaches after malware was slipped into a software update, is a sobering reminder that though updates are necessary, they are also always fraught with risk.  The [full impact](https://en.wikipedia.org/wiki/2020_United_States_federal_government_data_breach) of the attack, which is known to have affected computer systems within the U.S. Departments of Defense, State, Homeland Security, Treasury, Commerce, and Energy, is still to be tallied.
+
+Yet, ignoring updates is not a viable option either  As Uptane steering committee member Dr. Justin Cappos, an associate professor of computer science and engineering, explained in a December 20 article in [Yahoo Finance](https://au.finance.yahoo.com/news/why-russias-massive-cyberattack-is-especially-insidious-222912267.html?__s=p54njaazgqic1gqfruk3), nation state actors are gravitating to such attack targets because updating software is something system maintainers are “supposed to be doing.” In a sense, OEMs and suppliers find themselves between the proverbial rock and a hard place. But, even though updates are risky, he cautioned,”If you don’t apply software updates, you’re absolutely, definitely vulnerable because old software is vulnerable software.”
+
+For automotive OEMs,securing updates is much more complicated than for a conventional server based system, as the ECUs  must deal with limited memory, sometimes nonexistent storage, and the lack of a direct connection to the internet.
+
+## **What are the threats and why isn’t signing enough?**
+Increasingly, software updates for automotive ECUs are done using ["software-over-the-air”](https://ihsmarkit.com/research-analysis/remote-software-update-future-growth-business.html)or SOTA strategies, in which revised versions of programs are sent directly to computing units through broadcasts over the Internet. The strategy is growing in popularity because it is a much quicker and more cost-effective delivery mechanism than traditional methods, such as distributing flash drives or requiring customers to bring in their vehicles for servicing. Established OEMs are recognizing the potential of this strategy. In May 2019, GM In May 2019, GM [announced](https://www.theverge.com/2019/5/21/18633000/gm-ota-software-updates-digital-platform-reuss)  that the 2020 Cadillac CT5 sedan will feature over-the-air software update, and that such systems will be standard on all GM vehicles in the next four years. Likewise, Ford’s website  announced that it would begin equipping most redesigned vehicles in the U.S. with advanced over-the-air update (OTA) beginning in 2020, starting with the all-electric [Mustang Mach-E](https://media.ford.com/content/fordmedia/fna/us/en/news/2020/05/12/new-ford-over-the-air-updates-mustang-mach-e.html).
```

Comment:
```suggestion
Increasingly, software updates for automotive ECUs are done using ["software-over-the-air”](https://ihsmarkit.com/research-analysis/remote-software-update-future-growth-business.html) (SOTA) strategies, in which revised versions of programs are sent directly to computing units through broadcasts over the Internet. The strategy is growing in popularity because it is a much quicker and more cost-effective delivery mechanism than traditional methods, such as distributing flash drives or requiring customers to bring in their vehicles for servicing. Established OEMs are recognizing the potential of this strategy. In May 2019, GM [announced](https://www.theverge.com/2019/5/21/18633000/gm-ota-software-updates-digital-platform-reuss)  that the 2020 Cadillac CT5 sedan will feature an over-the-air software update mechanism, and that such systems will be standard on all GM vehicles in the next four years. Likewise, Ford’s website announced that it would begin equipping most redesigned vehicles in the U.S. with advanced over-the-air update mechanisms beginning in 2020, starting with the all-electric [Mustang Mach-E](https://media.ford.com/content/fordmedia/fna/us/en/news/2020/05/12/new-ford-over-the-air-updates-mustang-mach-e.html).
```